specs/tables.md

```diff
@@ -123,6 +123,9 @@ Proved by the bytecode circuit.
 
 Proved by the block circuit.
 
+__Note that a generalisation is done by storing the ChainId field in the block_table__
+__when it should indeed live inside of the tx_table.__
```

Comment:
I wouldn't say that the ChainId should live in the tx_table.  I think the ChainId should live in the chain configuration section, but we don't have that, so we put it in the block table as a generalization?

Reply:
Sounds nice! Will change it!